### ISSUE TITLE
Changing laravel property to laravel_cache  

### ISSUE THREAD

**Issue by adnkh** [spearmootz/node-laravel-session#10]
Hi,

Thanks for this interesting package,

As Laravel 5.5 (don't know exactly when it happened), I think `laravel` property should be replaced by `laravel_cache` in the `getSessionFromRedis`:

```
getSessionFromRedis: function (laravelSessionKey, redisConnection) {
        return new Promise(function (resolve, reject) {
            redisConnection.get('laravel_cache:' + laravelSessionKey, function (err, value) {
              console.log(value);
                if (err != null) return reject(err);

                return resolve(unserialize2(unserialize2(value)));
            });
        });

```

**Comment by spearmootz** [comment]
hey, thanks for reporting. let me investigate when this happened so i can add it to the README and verify the change.

**Comment by spearmootz** [comment]
@adnkh hey i published version 2.0.1 which allows you to change the sessionPrefix but it defaults to 'laravel_cache'

it took a while because i wanted to make sure the information you gave me was correct, which it was.

can you update and let me know if it works well for you?

**Comment by adnkh** [comment]
Awesome, It works now 💯 

thanks for the fast update 👍
